### ISSUE TITLE
Query param `?` separator not inserted in URL

### ISSUE THREAD

**Issue by jessecoleman** [Bewinxed/svetch#5]
Hi, just started playing around with this library, no configuration yet. But right off the bat I noticed that query params are not being separated from the URL path by a `?`

Example:
```
        const query = await svetch.get('/api/puzzles/daily/month', {
          query: { date: dateToString(d), mode: $mode }
        })
```

Result:
![image](https://github.com/user-attachments/assets/8cc04d67-4d4d-4b21-88ea-460d0d5b053d)

This can be fixed by adding a question mark to the first param of `svetch.get` but that breaks the types. Thanks for the cool library!

**Comment by jessecoleman** [comment]
https://github.com/Bewinxed/svetch/blob/49746008fff195118927df4322aeb868c39cdc4d/src/assets/client.ts#L123

I think on this line, it needs to be
```
		const url = `${baseUrl ?? ""}${updatedEndpoint}${query_params.size ? '?' : ''}${query_params.toString()}`;
```
Or something to that effect.

**Comment by Bewinxed** [comment]
Thank you, fixed in 2.1.45

`npx ***@***.***`

**Comment by jessecoleman** [comment]
Thanks, seems to be working now!
